test/regression/pom.xml

```diff
@@ -777,5 +786,176 @@
+                                    <reportsDirectory>${project.build.directory}/surefire-reports-ssl</reportsDirectory>
+                                    <environmentVariables>
+                                        <BYTEMAN_HOME></BYTEMAN_HOME>
+                                    </environmentVariables>
+                                </configuration>
+                            </execution>
+
+                            <execution>
+                                <id>imr</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <useManifestOnlyJar>false</useManifestOnlyJar>
+                                    <forkCount>${surefire.forkCount}</forkCount>
+                                    <reuseForks>false</reuseForks>
+                                    <argLine>${jacoco.agent.argLine} -Djacorb.test.home=${basedir} -Djacorb.test.verbose=${jacorb.test.verbose} -Djacorb.test.imr=true -Djdk.attach.allowAttachSelf=true</argLine>
+                                    <reportsDirectory>${project.build.directory}/surefire-reports-imr</reportsDirectory>
+                                    <environmentVariables>
+                                        <BYTEMAN_HOME></BYTEMAN_HOME>
+                                    </environmentVariables>
+                                    <groups>org.jacorb.test.harness.ClientServerCategory</groups>
+                                    <excludedGroups>org.jacorb.test.harness.IMRExcludedClientServerCategory,org.jacorb.test.harness.FixedPortCategory</excludedGroups>
+                                </configuration>
+                            </execution>
+
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+		</profile>
+		
+		<profile>
+			<id>test-use-jboss-rmi</id>
+            <activation>
+                <property>
+                    <name>useRMI</name>
+                    <value>jboss</value>
+                </property>
+            </activation>
+			<dependencies>
+				<dependency>
+				    <groupId>org.jboss.spec.javax.rmi</groupId>
+				    <artifactId>jboss-rmi-api_1.0_spec</artifactId>
+				    <version>1.0.6.Final</version>
+				</dependency>
+			</dependencies>
+		</profile>
+		
+		<profile>
+			<id>test-use-rmistub</id>
+            <activation>
+                <property>
+                    <name>useRMI</name>
+                    <value>stubs</value>
+                </property>
+            </activation>
+			<dependencies>
+	            <dependency>
+	                <groupId>org.jacorb</groupId>
+	                <artifactId>jacorb-rmistub</artifactId>
+	                <version>3.10-SNAPSHOT</version>
```

Comment:
Should be ${project.version} or even better remove the version and ensure `jacorb-rmistub` is added to the dependencyManagement at the top level?

Reply:
Replaced with #{project.version}, also some more tab to spaces